### PR TITLE
correcting typos in type database

### PR DESCRIPTION
**Pull request by jean-quilbeuf**

### CODE REVIEW

File: libr/anal/d/types [hunk 1]
```diff
@@ -145,11 +145,6 @@ func.malloc.args=1
 func.malloc.arg.0=size_t,size
 func.malloc.ret= void *
 
-xmalloc=func
```
Comment (radare) [inline libr/anal/d/types]: Why remove it?

Comment (jean-quilbeuf) [inline libr/anal/d/types]: It was a mistake, sorry for your time.

**Comment by jean-quilbeuf** [comment]
Sorry, that is my first pull request.

File: libr/anal/d/types [hunk 2]
```diff
@@ -146,9 +146,9 @@ func.malloc.arg.0=size_t,size
 func.malloc.ret= void *
 
 xmalloc=func
-func.xmalloc.args=1
-func.xmalloc.arg.0=size_t,size
-func.xmalloc.ret=void *
+func.malloc.args=1
```
Comment (radare) [inline libr/anal/d/types]: this is xmalloc, not malloc, see line above

Comment (radare) [inline libr/anal/d/types]: malloc is already defined in the block above

File: libr/anal/d/types [hunk 3]
```diff
@@ -146,9 +146,9 @@ func.malloc.arg.0=size_t,size
...
+func.malloc.arg.0=size_t,size
+func.malloc.ret= void *
```
Comment (radare) [inline libr/anal/d/types]: remove that space after the =

File: libr/anal/d/types (hunk 2 above)
Comment (radare) [inline libr/anal/d/types]: please change this malloc to xmalloc again

**Comment by radare** [comment]
ping @jean-quilbeuf

**Comment by radare** [comment]
tired of waiting i have fixed this by myself.

thanks!

**Pull request closed without merging**